src/mappings/handlers/evmHandlers.ts

```diff
@@ -9,13 +11,314 @@ import { InvestorTransactionData, InvestorTransactionService } from '../services
+    pile: [{ address: '0xc2a75288cf644b068fC1e765586c170f2529a87F' }],
+  },
+  {
+    id: '0x0ced6166873038ac0cc688e7e6d19e2cbe251bf0',
+    shortName: 'Bling Series 1',
+    startBlock: 11196893,
+    navFeed: [{ address: '0x1621b607a62dac0dc2e4044ff1235a30f135cbd2', startBlock: 11197170 }],
+    reserve: [{ address: '0x932344ba99bf34035b4bc25cbd98f912ebc60371', startBlock: 11197219 }],
+    shelf: [{ address: '0xCFad06aDAcf221f8119995c8bCa25184A6b5A268' }],
+    pile: [{ address: '0x05739C677286d38CcBF0FfC8f9cdbD45904B47Fd' }],
+  },
+  {
+    id: '0x235893bf9695f68a922dac055598401d832b538b',
+    shortName: 'Pezesha 1',
+    startBlock: 12453056,
+    navFeed: [{ address: '0xd9b2471f5c7494254b8d52f4ab3146e747abc9ab', startBlock: 12453087 }],
+    reserve: [{ address: '0xb12e705733042610174ed22f6726d26db12dbdfe', startBlock: 12703288 }],
+    shelf: [{ address: '0x4Ca7049E61629407a7E829564C1Dd2538d70182C' }],
+    pile: [{ address: '0xAAEaCfcCc3d3249f125Ba0644495560309C266cB' }],
+  },
+  {
+    id: '0x3170d353772ed68676044f8b76f0641b2cba084e',
+    shortName: 'Fortunafi 2',
+    startBlock: 13512637,
+    navFeed: [{ address: '0x3263b3a84543abfceca35ab1b02c6ab4f1e66b17', startBlock: 13512663 }],
+    reserve: [{ address: '0x101e8f7598639bf3d1320ac9999977a0678aa186', startBlock: 13513359 }],
+    shelf: [{ address: '0x720E872De93F3416bD0d43d3ACA6891f26D60797' }],
+    pile: [{ address: '0x2bD53Bd2341329e0cf36c55380A77a3E457028Ca' }],
+  },
+  {
+    id: '0x3b03863bd553c4ce07eabf2278016533451c9101',
+    shortName: 'Cauris Global Fintech 1',
+    startBlock: 13397601,
+    navFeed: [{ address: '0x431548e6f14134b7e2955c4f2d42054f7588afce', startBlock: 13397666 }],
+    reserve: [{ address: '0xb15ccb81f633b98845a0f6c91a89483298a41cc3', startBlock: 13402984 }],
+    shelf: [{ address: '0x5621d692DEEea943C0c694dC3000E32d3CE28bA7' }],
+    pile: [{ address: '0x696866ea53C0f521C24a28D92fAbD2c559182D38' }],
+  },
+  {
+    id: '0x3d167bd08f762fd391694c67b5e6af0868c45538',
+    shortName: 'GIG Pool',
+    startBlock: 12901666,
+    navFeed: [{ address: '0x468eb2408c6f24662a291892550952eb0d70b707', startBlock: 12901939 }],
+    reserve: [{ address: '0x1794a4b29ff2ecdc044ad5d4972fa118d4c121b9', startBlock: 12902086 }],
+    shelf: [{ address: '0x661f03AcE6Bd3087201503541ac7b0Cb1185d673' }],
+    pile: [{ address: '0x9E39e0130558cd9A01C1e3c7b2c3803baCb59616' }],
+  },
+  {
+    id: '0x4597f91cc06687bdb74147c80c097a79358ed29b',
+    shortName: 'BT1',
+    startBlock: 16041343,
+    navFeed: [{ address: '0x479506bff98b18d62e62862a02a55047ca6583fa', startBlock: 16041344 }],
+    reserve: [{ address: '0x50f8a1cdedd9aef2901c7ec0859d8adbf61b1263', startBlock: 16041344 }],
+    shelf: [{ address: '0x35Cbc4F6de5333DAFb372200591E69b68B2A5FbF' }],
+    pile: [{ address: '0x62E6225d9DbFa9C5f09ccB43304F60a0a7dDeb7A' }],
+  },
+  {
+    id: '0x4b6ca198d257d755a5275648d471fe09931b764a',
+    shortName: 'Fortunafi 1',
+    startBlock: 11282587,
+    navFeed: [
```

Comment:
This is a really clean format, nice work!

I would move the `pools` array here to `config.ts` though